Makefile

```diff
@@ -64,6 +64,13 @@ else ifeq ($(UNAME),Darwin)
 LIBEXT=dylib
 CFLAGS += -m$(ARCH) -I /opt/libjpeg-turbo/include -I /usr/local/opt/jpeg-turbo/include -I /usr/local/include -I /usr/local/opt/libvorbis/include -I /usr/local/opt/openal-soft/include -Dopenal_soft  -DGL_SILENCE_DEPRECATION
 LFLAGS += -Wl,-export_dynamic -L/usr/local/lib
+
+ifneq ($(origin SDK), undefined)
```

Comment:
can you instead use  ifdef with variable OSX_SDK ?

Reply:
Okay, updated